src/main/xerus/monstercat/Settings.kt

```diff
@@ -42,6 +42,42 @@ object Settings : SettingsNode("xerus/monsterutilities") {
 	
 	val FILENAMEPATTERN = create("updatePattern", "MonsterUtilities-%version%.jar")
 	
+	enum class ConnectionSpeed(val maxConnections: Int) {
+		DIALUP(5) {
+			override fun toString() = "Dial-up (100 kb/s)"
+		},
+		ADSL(30) {
+			override fun toString() = "ADSL / 3G (10 Mb/s)"
+		},
+		CABLE(150) {
+			override fun toString() = "Cable / 4G (100 Mb/s)"
+		},
+		FIBER(300) {
+			override fun toString() = "Fiber (300+ Mb/s)"
+		};
+		
+		companion object {
+			fun findFromValue(maxConnections: Int): ConnectionSpeed {
+				ConnectionSpeed.values().forEach {
+					if (it.maxConnections == maxConnections)
```

Comment:
why don't you simply do `it.maxConnections >= maxConnections`, in case we change the value?

Reply:
We could but I don't see any real interest.